tasks/go.py

```diff
@@ -47,6 +52,13 @@
 # Packages that need go:generate
 GO_GENERATE_TARGETS = ["./pkg/status", "./cmd/agent/gui"]
 
+# LICENSE files
+LICENSE_FILES = ["LICENSE", "LICENSE.md"]
```

Comment:
This feels too restrictive, compare this with e.g. the `wwhrd` list https://github.com/frapposelli/wwhrd/blob/a4048a3e900ae413910a8477d7c8c2cf9eb9fc3a/walker.go#L22-L65